packages/app-desktop/gui/ItemList.jsx

```diff
@@ -69,7 +69,8 @@ class ItemList extends React.Component {
 		} else {
 			scrollTop = this.props.itemHeight * itemIndex - (this.visibleItemCount() - 1) * this.props.itemHeight;
 		}
-
+		
+		if (!this.props.visible) scrollTop = 0;
```

Comment:
Doesn't it mean that if the user hides the note list, then show it again, the scroll position will have changed? That would be a regression as currently the position is saved in this case.